apache-solrj/pom.xml

```diff
@@ -0,0 +1,38 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>apache-solrj</groupId>
+  <artifactId>apache-solrj</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  
+  <properties>
+  	<junit.version>4.12</junit.version>
+  </properties>
+  
+  <dependencies>
+    <dependency>
+		<groupId>org.apache.solr</groupId>
+  		<artifactId>solr-solrj</artifactId>
+  		<version>6.4.0</version>
+	</dependency>
+	<dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>${junit.version}</version>
+            <scope>test</scope>
+    </dependency>
+  </dependencies>
+  
+  <build>
+    <sourceDirectory>src</sourceDirectory>
+    <plugins>
+      <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
missing the groupId tag for the maven-compiler-plugin: its value should be org.apache.maven.plugins (see one of the other projects for example)